### DISCUSSION TITLE
HTML to powerpoint example

### DISCUSSION THREAD

**Discussion by Strawberry0215** [gitbrent/PptxGenJS#1098] [General]
This function is not working for me, can you please help me here
  pptx.tableToSlides("TableKanban", {
  addImage: { path: "https://upload.wikimedia.org/wikipedia/en/a/a9/Example.jpg", x: 10, y: 0.5, w: 1.2, h: 0.75 }
});

![image](https://user-images.githubusercontent.com/103405409/162687508-cfbb7064-50fe-4e7d-8b05-df604696d07b.png)
The error says:

the error says:  pptxgen.min.js:2 ERROR: addImage() requires either 'data' or 'path' parameter!

**Comment by gitbrent** [comment]
Thanks for asking about this problem @Strawberry0215 

Issue #1103 has been completed to resolve this error message.

```javascript
let pptx = new PptxGenJS();
objOpts.addImage = { image: { data: CHECKMARK_GRN }, options: { x: 12.67, y: 0.0, w: 0.67, h: 0.67 } };
pptx.tableToSlides("tabAutoPaging", objOpts);
```

![Screen Shot 2022-05-01 at 13 38 41](https://user-images.githubusercontent.com/7218970/166159950-f546d3ec-feef-4a92-bb40-8c019609ce7f.png)
